go.mod

```diff
@@ -162,7 +164,7 @@ require (
 	go.etcd.io/etcd/client/v3 v3.5.10 // indirect
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.42.0 // indirect
 	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace v0.42.0 // indirect
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0 // indirect
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.46.0 // indirect
```

Comment:
Is this change a mistake? This dep already has a replace directive at the top of the file. Also, we don't usually manually change the indirects, opting to let the go tooling manage the indirects for us.

Reply:
Not sure. It's the result of a `go mod tidy`, so I assumed it was correct.

Reply:
Strange, if I add the diffs that you made above (not including this one) and then run `go mod tidy` then it does not change any indirect dep lines. That what I would expect, because you only added a `replace`, and a `replace` should never impact the indirects listed in the go.mod (except for having the effect of replacing them at compile time, of course, but not editing them in the file).